docs/lti.md

```diff
@@ -2,9 +2,135 @@
 
 Marsha can act as an LTI provider and supports version 1.0 of the standard.
 
-For developers' reference, here are examples of LTI launch requests sent by the most common LMS. If your LMS is missing from this list, please feel free to submit a Pull Request to add it to the list and we will make sure Marsha supports it.
+In order to connect an LTI consumer site (such as Moodle or Open edX) a matching passport needs to be created in
+Marsha.
 
-## Launch request sent by Open edX
+## LTI Deep Linking
+
+LTI content can be selected in Moodle through Deep Linking.
```

Comment:
We should also say that this feature can be use with any LMS with LTI capabilities. This is not dedicated to moodle.